Traceback (most recent call last):
  File "C:\Users\Antho\temp.py", line 3, in <module>
    from transformers import pipeline
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\site-packages\transformers\__init__.py", line 26, in <module>
    from . import dependency_versions_check
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\site-packages\transformers\dependency_versions_check.py", line 16, in <module>
    from .utils.versions import require_version, require_version_core
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\site-packages\transformers\utils\__init__.py", line 60, in <module>
    from .hub import (
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\site-packages\transformers\utils\hub.py", line 32, in <module>
    from huggingface_hub import (
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\site-packages\huggingface_hub\__init__.py", line 322, in __getattr__
    submod = importlib.import_module(submod_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Antho\AppData\Local\Programs\Python\Python311\Lib\site-packages\huggingface_hub\file_download.py", line 21, in <module>
    from filelock import FileLock
ModuleNotFoundError: No module named 'filelock'
